browser/css/jssidebar.css

```diff
@@ -350,13 +357,21 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
 	width: var(--btn-size-s);
 	height: var(--btn-size-s);
 }
+#NavigatorDeck .ui-treeview-entry.selected:not(.ui-treeview-expandable) {
+	background-color: var(--color-background-dark);
+	color: var(--color-primary);
+}
+#NavigatorDeck .ui-treeview-entry.selected:not(.ui-treeview-expandable)::before {
```

Comment:
we cannot use :not(.ui-treeview-expandable) since this would make the objects with children without highlight

I need an additional CSS class to be add to `.ui-treeview-entry`  that have icon inside (`img.ui-listview-icon`)

Reply:
can anyone do this for me please? CC:  @gokaysatir @Darshan-upadhyay1110 ?

Reply:
@pedropintosilva Sure i will check this